src/app/hierarchical-grid/hierarchical-grid-collapsible-column-groups/hierarchical-grid-collapsible-column-groups.component.html

```diff
@@ -34,10 +34,10 @@
                 <igx-row-island [key]="'Orders'" [autoGenerate]="false">
```

Comment:
Set row island moving property to 'true', so the columns in the child grids to be movable as before